### ISSUE TITLE
bee run "Build failed" after change any .go file

### ISSUE THREAD

**Issue by lexa-uw** [beego/bee#182]
Example:

``` bash
$ cd ~/go/src
$ bee new test
...
$ cd ~/go/src/test
$ bee run
bee   :1.4.1
beego :1.6.1
Go    :go version go1.6 linux/amd64

2016/03/21 15:20:05 [INFO] Uses 'test' as 'appname'
2016/03/21 15:20:05 [INFO] Initializing watcher...
2016/03/21 15:20:05 [TRAC] Directory(/home/vagrant/go/src/test/controllers)
2016/03/21 15:20:05 [TRAC] Directory(/home/vagrant/go/src/test)
2016/03/21 15:20:05 [TRAC] Directory(/home/vagrant/go/src/test/routers)
2016/03/21 15:20:05 [TRAC] Directory(/home/vagrant/go/src/test/tests)
2016/03/21 15:20:05 [INFO] Start building...
2016/03/21 15:20:08 [SUCC] Build was successful
2016/03/21 15:20:08 [INFO] Restarting test ...
2016/03/21 15:20:08 [INFO] ./test is running...
2016/03/21 15:20:08 [asm_amd64.s:1998][I] http server Running on :8080
```

And run it in another console:

``` bash
echo " " >> controllers/default.go
```

Result in previous console:

``` bash
2016/03/21 15:21:13 [EVEN] "/home/vagrant/go/src/test/controllers/default.go"
: MODIFY
2016/03/21 15:21:14 [INFO] Start building...
go install test: open test: text file busy
2016/03/21 15:21:16 [ERRO] ============== Build failed ===================
```

**Comment by SydOps** [comment]
Seems the problem has been fixed in bee version v1.5.2

**Comment by cheanizer** [comment]
me using v1.10.0 with docker and windows 10. 

and got this error too, 
`| ___ \
| |_/ /  ___   ___
| ___ \ / _ \ / _ \
| |_/ /|  __/|  __/
\____/  \___| \___| v1.10.0
2020/03/01 16:58:56 INFO     ▶ 0001 Using 'MyApp' as 'appname'
2020/03/01 16:58:56 INFO     ▶ 0002 Initializing watcher...
MyApp
2020/03/01 16:59:02 SUCCESS  ▶ 0003 Built Successfully!
2020/03/01 16:59:02 INFO     ▶ 0004 Restarting 'MathApp'...
2020/03/01 16:59:02 SUCCESS  ▶ 0005 './MyApp' is running...
2020/03/01 16:59:02.754 [I] [asm_amd64.s:1333]  http server Running on http://:8080
MyApp
2020/03/01 16:59:21 ERROR    ▶ 0006 Failed to build the application: go build MyApp: open MathApp: text file busy`

look like a file system issue isn't it?
